.github/workflows/release.yml

```
@@ -1,29 +1,29 @@
 name: Release
 
 on:
-  workflow_dispatch:
-    inputs:
-      versionInc:
-        description: 'Version increment'
-        required: true
-        type: choice
-        options:
-        - patch
-        - minor
-        - major
+  push:
+    branches:
+      - main
+
+concurrency: ${{ github.workflow }}-${{ github.ref }}
 
 jobs:
   release:
+    name: Release
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - name: Checkout Repo
+        uses: actions/checkout@v2
```

In the previous config, we also defined a fetch-depth -- should we include this one here as well?

```suggestion
        uses: actions/checkout@v2
        with:
          fetch-depth: 0
```